frontend/src/app/spot/styles/sass/components/modal.sass

```diff
@@ -38,7 +38,10 @@
     position: relative
     align-items: center
 
-    .spot-icon
+    + .spot-divider
+      margin-top: $spot-spacing-1
```

Comment:
this should not be needed? the top margin of the divider is defined in `divider.sass`

Reply:
We [removed this](https://github.com/opf/openproject/pull/11246/files#diff-64ce798fd9b6656777121aa833252cfe12e3d84a026c30eb47461ad38f88cc85L2) because it breaks the BEM assumption that blocks never have margins.

Instead we added them to the context right here (spot-modal) and [in spot-container:](https://github.com/opf/openproject/pull/11246/files#diff-2246b4e29d74b4ce0bee934476e29eaed67eb2c59c08bdec4e8a1ce4e2f527d5R13)

Reply:
I actually think `spot-divider` should maybe always be an `hr` instead of a `div`, but that's a quick change we can make any time.